app/api/sessions.py

```diff
@@ -86,7 +86,7 @@ def query(self, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        query_ = self.session.query(Session)
+        query_ = self.session.query(Session).filter(Session.state == 'accepted')
```

Comment:
Organizer should be able to see all in selected places. Please check it works. Also, attendee should not be able to see pending ones even after using filtering. Please check this